friends/friendModel.js

```diff
@@ -0,0 +1,34 @@
+const mongoose = require('mongoose');
+
+// Schema
+// {
+//     firstName: "Jane", // String, required
+//     lastName: "Doe",  // String, required
+//     age: 18, // Number, required, should be an integer between 1 and 120
+//     createdOn: Mon Aug 14 2017 12:50:16 GMT-0700 (PDT) // Date, required, defaults to current date
+//   }
+
+const FriendSchema = new mongoose.Schema({
+    firstName: {
+        type: String, 
+        required: true,
+    }, 
+    lastName: {
+        type: String, 
+        required: true
+    },
+    age: {
+        type: Number,
+        required: true,
+        minimum: 1, 
```

Comment:
Nice! In theory the min/max shouldn't have any effect, because your Controller is checking for numbers outside this range and kicking back an error if the number is too low/high. But this is still a great exercise in using validators and thinking about your mongoose schema.

Reply:
Good point, thanks Ellen!